simpleimage/make_rootfs.sh

```diff
@@ -327,7 +354,9 @@ EOF
 			DEBUSERPW=ubuntu
 			EXTRADEBS="software-properties-common zram-config ubuntu-minimal"
 			ADDPPACMD="apt-add-repository -y ppa:longsleep/ubuntu-pine64-flavour-makers"
-			DISPTOOLCMD="apt-get -y install sunxi-disp-tool"
+			if [ ! "$LCD" = "-lcd" ]
```

Comment:
Please change this comparison to check for a "1" variable which is set on the top of the script based on commandline parameter. Rest looks good. Also, please do not open new pull requests all the time. Use the existing one thanks!